pages/map.tsx

```diff
@@ -0,0 +1,27 @@
+import Head from "next/head";
+import Header from "../components/Header";
+import dynamic from "next/dynamic";
+import Footer from "../components/Footer";
+import { ArtistInterface } from "../lib/ArtistClass";
+
+type MapProps = {
+  artists: ArtistInterface[];
+};
+
+export default function MapPage({ artists }: MapProps) {
+  const Map = dynamic(() => import("../components/Map"), {
+    loading: () => <p>A map is loading</p>,
+    ssr: false,
+  });
+  return (
+    <>
+      <Head>
+        <title>Wannado</title>
+        <link rel="shortcut icon" href="/favicon.ico" />
+      </Head>
```

Comment:
Head einbinden für den Title steht bei mir auch noch auf der Liste - ich frag mich gerade, ob du das in der _app.js machen kannst, sodass es für alle Seiten gilt?

Reply:
Gute Frage!